.nuspec/Microsoft.Maui.Controls.SingleProject.targets

```diff
@@ -104,6 +104,7 @@
     <ProjectCapability Include="XamarinStaticLaunchProfiles" Condition=" '$(VisualStudioVersion)' != '' and '$(VisualStudioVersion)' &lt; '17.0' " />
     <!-- Otherwise define LaunchProfilesGroupByPlatformFilters by default -->
     <ProjectCapability Include="LaunchProfilesGroupByPlatformFilters" Condition=" '$(VisualStudioVersion)' == '' or '$(VisualStudioVersion)' &gt;= '17.0' " />
+    <ProjectCapability Include="SingleTargetBuildForStartupProjects" />
```

Comment:
Two questions:

1. Did we test this works in the IDE? I worry that the capability won't be there, the way workloads work. It seems like the "outer" build doesn't load any workload MSBuild targets.
1. Should we also put this in each platform workload? Like you could do a `<TargetFrameworks>net6.0-android;net6.0-ios;net6.0-windows</TargetFrameworks>` and not be using MAUI? Like a multi-targeted SkiaSharp app or something.

Reply:
To test this try:
```xml
<ItemGroup>
  <ProjectCapability Include="DiagnoseCapabilities" />
```
And it should list them all in the solution tree.

https://www.vsixcookbook.com/tips/projects.html